docs/source/api/apollo-server.mdx

```diff
@@ -387,14 +361,13 @@ You can also manually call `stop()` in other contexts. Note that `stop()` is asy
 <tr>
 <td>
 
-##### `debug`
+##### `includeStacktraceInErrorResponses`
 
-`Boolean`
+`boolean`
 </td>
 
 <td>
-
-If `true`, stack traces are included in GraphQL responses when errors occur, and some extra information is logged via the `logger` at the `debug` level.
+If `true`, stack traces are included in GraphQL responses when errors occur. 
```

Comment:
`true` isn't showing up properly, I think maybe you need a blank line before this

Reply:
Great catch!